components/CropSelector/CropSelector.vue

```diff
@@ -97,9 +99,28 @@ export default {
        */
       this.$emit('valid', event);
     },
-    handleAddClicked() {
+    async handleAddClicked(newCrop) {
+      // Clear the cached
```

Comment:
I think one comment describing what this method is doing and why would be more useful than the individual line comments describing what the lines are doing.